### PR TITLE
Add native props to Breadcrumb

### LINKED ISSUE #8151: Breadcrumb not supporting top level native props
### Environment Information 
 
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)

### Please provide a reproduction of the bug in a codepen:
 

The background for this issue is that I need change the breadcrumb from being a navigation landmark (since this can be confusing if you are using several breadcrumbs on the same page, or if you have a breadcrumb without any links.)
I've tried changing this by setting role="region" on the component but this has no effect on the top level element. It appears the breadcrumb is only extending `React.ClassAttributes` rather than `React.HTMLAttributes`.
 
#### Actual behavior:
 
role="navigation" for top level div
 
#### Expected behavior:
 
role="region" for top level div

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)

### PR DESCRIPTION
**Pull request by leddie24**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #8151
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Adds native props support to Breadcrumb

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8152)

### CODE REVIEW

**Comment by aneeshack4** [comment]
@leddie24 Looks like there are some build errors

**Comment by leddie24** [comment]
@cliffkoh added a test, can you please review at your earliest convenience?

**Comment by cliffkoh** [comment]
@leddie24 please hold off on merging while I investigate what is happening with automerge :)

File: common/changes/office-ui-fabric-react/edwl-2019-02-addNativePropsBreadcrumb_2019-02-28-19-57.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Breadcrumb: Add native props support",
+      "type": "patch"
```
Comment (dzearing) [inline common/changes/office-ui-fabric-react/edwl-2019-02-addNativePropsBreadcrumb_2019-02-28-19-57.json]: This is a minor, you're adding a feature.

**Pull request merged**